packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1696,6 +1746,79 @@ export const FIREBIRDONESWAP_POLYGON_INFOS: { [name: string]: CurveInfo } = {
     },
 };
 
+export const MOBIUSMONEY_CELO_INFOS: { [name: string]: CurveInfo } = {
```

Comment:
If you have a link on where you found these pools I'd suggest commenting it above so we know where to go if we get wind of a new pool.

Reply:
yup ill link it in the description